lib/modules/manager/pip_requirements/extract.spec.ts

```diff
@@ -209,7 +209,14 @@ describe('modules/manager/pip_requirements/extract', () => {
             depName: 'python-pip-setup-test',
             currentValue: 'v0.9.0',
             currentVersion: 'v0.9.0',
-            packageName: 'peter@github.com:rwxd/python-pip-setup-test.git',
```

Comment:
is this invalid? if it's valid then please leave as test.

Reply:
the `:` in the packageName is not read correctly if trying to connect over https. hence I have modified the logic and the test